tests/run-make/lto-avoid-object-duplication/rmake.rs

```diff
@@ -0,0 +1,38 @@
+// ignore-tidy-tab
```

Comment:
Question: is `ignore-tidy-tab` needed in this file for some reason? I can't immediately see why it's needed.

Reply:
It is, because of this line right here:
`    // 0000000000000000 g     F .text.issue64153_test_function	00000023 issue64153_test_function`
It's demonstrating that g and F may have more than one space in between, and tidy complains about that.

Reply:
Oh hecc, thanks